libs/core/src/scss/base/_line-clamp.scss

```diff
@@ -36,7 +36,7 @@
 // See https://caniuse.com/mdn-css_properties_-webkit-line-clamp
 @supports (-webkit-line-clamp: 1) {
   .kirby-line-clamp {
-    display: -webkit-box;
+    display: box;
```

Comment:
I don't believe `box` is a valid value for the `display` property.

What's even worse, line-clamp will not work in any browsers without  `display: -webkit-box`.

Do you know what setting/configuration triggers this change? We _do_ have a `.browserslistrc` file. Maybe stylelint/prettier uses it to determine which corrections to apply...?

Reply:
```suggestion
    display: -webkit-box;
```

Reply:
Nice. This is caused by the https://stylelint.io/user-guide/rules/list/value-no-vendor-prefix/ rule. 
Should we customise the rule to ignore specific properties, or disable it entirely?

Reply:
As per your other comment, I can see that stylelint integrates with Autoprefixer by default, so maybe it should just be disabled for this specific line, if it is still needed?

Reply:
Done!